From 98243492079e54e7f27e8213895fb0063b56242c Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Mon, 5 Jan 2015 11:53:06 -0500
Subject: [redhat] kabi: Adding symbol pci_get_subsys (include/linux/pci.h)

Message-id: <024ba283057c89be483f6274b783c4621e33990c.1420451770.git.poros@redhat.com>
Patchwork-id: 102442
O-Subject: [PATCH RHEL7.1 09/24] kabi: Adding symbol pci_get_subsys (include/linux/pci.h)
Bugzilla: 1085963
RH-Acked-by: Myron Stowe <mstowe@redhat.com>

Bugzilla: 1085963
    https://bugzilla.redhat.com/show_bug.cgi?id=1085963

    Description:
    ============
    Needed to search for cache module and other pci devices that
    the driver supports.
    Used in hpvsa.ko, hpdsa.ko

Symbol History
==============
 Request:
 ========
 [PATCH RHEL7 BZ1070457 09/18] kabi: Adding symbol pci_get_subsys (drivers/pci/search.c)
 Jiri Olsa <jolsa@redhat.com> (Thu, 27 Feb 2014 10:03:23 +0100)
 Description:
 ============
 used by raidstack libary to get pci bus information

 Reviews:
 ========
 N/A

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/pci_get_subsys
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64le/pci_get_subsys
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/pci_get_subsys
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/pci_get_subsys

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/pci_get_subsys b/redhat/kabi/kabi-module/kabi_ppc64/pci_get_subsys
new file mode 100644
index 0000000..9698308
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/pci_get_subsys
@@ -0,0 +1,2 @@
+#1-
+0xb96fe991 pci_get_subsys vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_ppc64le/pci_get_subsys b/redhat/kabi/kabi-module/kabi_ppc64le/pci_get_subsys
new file mode 100644
index 0000000..7895b48
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64le/pci_get_subsys
@@ -0,0 +1,2 @@
+#1-
+0x114a6204 pci_get_subsys vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/pci_get_subsys b/redhat/kabi/kabi-module/kabi_s390x/pci_get_subsys
new file mode 100644
index 0000000..4c87d1f
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/pci_get_subsys
@@ -0,0 +1,2 @@
+#1-
+0xc0c5e0b1 pci_get_subsys vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/pci_get_subsys b/redhat/kabi/kabi-module/kabi_x86_64/pci_get_subsys
new file mode 100644
index 0000000..815257e
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/pci_get_subsys
@@ -0,0 +1,2 @@
+#1-
+0xbe872f4b pci_get_subsys vmlinux EXPORT_SYMBOL
-- 
1.7.1